### ISSUE TITLE
CLI migrate apply failed: failed parsing json: invalid character 'I' 

### ISSUE THREAD

**Issue by jcmoore0** [hasura/graphql-engine#3873]
Hi Team-  I am getting an error when doing a migration.
I don't have any error logs on server.
v1.1.0-beta.3
```
> hasura migrate status  (works properly)
> hasura migrate apply
FATA[0012] apply failed: failed parsing json: invalid character 'I' looking for beginning of value; response from API: Internal Server Error
>  hasura migrate status  (never completes....spinning)
  restart hasura server
>  hasura migrate status (works properly)
```
I repeated this sequence several times.

Thanks,
John

**Comment by shahidhk** [comment]
@jcmoore0 What is the CLI version?

How long does it take before this error happens? Can you also send us the server logs from this time?

**Comment by jcmoore0** [comment]
Hi @shahidhk -  I recommend closing this as  I am no longer able to reproduce this condition and cannot troubleshoot further.

FYI
Hasure & CLI version: 1.1.0-beta.3
There were never any server logs related to this event.
The client side error came after a few ~5 seconds.
Last week, I was able to reproduce the above errors repeatedly on 2 servers.
On one server I truncated hdb_catalog.schema_migrations, but it didn't resolve the issue.
Today, migrations work on both servers.

Thanks for your help.
John
